tests/ResultPrinter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yiiunit;
+
+/**
+ * IsOneOfAssert asserts that the value is one of the expected values.
```

Comment:
phpdoc looks wrong.

Reply:
phpunit issue should be referenced. It's not clear about the purpose of the printer from the code itself.

Reply:
Actually, it's not a PHPUnit issue.
PHP have changed `session_cache_limiter()` so it does not work when headers are already sent.
https://github.com/php/php-src/blob/php-7.2.0RC4/UPGRADING#L89

So our tests are broken. Maybe it will be better to mock `session_cache_limiter()` call instead of changing printing output